source/fundamentals/connection/network-compression.txt

```diff
@@ -0,0 +1,91 @@
+.. _compression:
+.. _network-compression:
+
+===================
+Network Compression
+===================
+
+You can enable a driver option to compress messages and therefore reduce
```

Comment:
comment: I think "therefore" is unnecessary here.
```suggestion
You can enable a driver option to compress messages and reduce
```

Reply:
I think it's slightly less clear that it's a consequence by using the conjunction "and" by itself. Alternatively, I could lower the word count and maintain that it's a consequence "compress messages which reduces", but that changes the tense to past continuous and I think we prefer simple past. Will try it out anyway.

Reply:
I thought it out of place since I usually associate conjunctive adverbs with commas, i.e. `You can enable a driver option to compress messages and, therefore, reduce...

I don't think this is important to wordsmith though.

Reply:
I think the style guide also mentions not using commas to break the flow, so I'm going to use the alternative mentioned to change that results in past continuous tense.